### PR TITLE
Add com.github.tenderowl.frog

### PR DESCRIPTION
**Pull request by amka**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.tenderowl.frog.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{
+    "app-id" : "com.github.tenderowl.frog",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
```
Comment (hfiguiere) [inline com.github.tenderowl.frog.json:4]: there is no `master` runtime on flathub.

File: com.github.tenderowl.frog.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                    "type" : "git",
+                    "url" : "https://github.com/tenderowl/frog.git",
+                    "tag": "0.3.0"
+                }
+            ]
+        }
+    ],
+    "build-options" : {
+        "env" : {        }
+    }
```
Comment (hfiguiere) [inline com.github.tenderowl.frog.json:98]:
```suggestion
    ]
```

File: leptonica.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (hfiguiere) [inline leptonica.json:1]: there is no point havimng this in a separate manifest file

File: libportal.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "libportal",
```
Comment (hfiguiere) [inline libportal.json:2]: there is no point havimng this in a separate manifest file

File: tesseract.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (hfiguiere) [inline tesseract.json:1]: there is no point havimng this in a separate manifest file

File: zbar.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (hfiguiere) [inline zbar.json:1]: there is no point havimng this in a separate manifest file

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build com.github.tenderowl.frog

File: com.github.tenderowl.frog.json [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        "leptonica.json",
+        "tesseract.json",
+        "libportal.json",
+        "zbar.json",
```
Comment (hfiguiere) [inline com.github.tenderowl.frog.json:28]: the content of these is missing now.

Comment (amka) [reply com.github.tenderowl.frog.json:28]: That's why I put them inside this repo. So, does it mean I should avoid putting any other files here, but put all the modules inside manifest instead?

Comment (hfiguiere) [reply com.github.tenderowl.frog.json]: that's exactly what I meant

**Comment by hfiguiere** [comment]
bot, build com.github.tenderowl.frog

**Comment by amka** [comment]
Removed invalid screenshot from appdata.

**Comment by amka** [comment]
bot, build com.github.tenderowl.frog

**Comment by barthalion** [comment]
Something is not quite right on my computer:

```
[bpiotrowski@risuku ~]$ flatpak run com.github.tenderowl.frog

** (frog:2): WARNING **: 08:27:46.670: Failed to load shared library 'libgranite.so.6.2.0' referenced by the typelib: libgranite.so.6.2.0: cannot open shared object file: No such file or directory
Traceback (most recent call last):
  File "/app/bin/frog", line 51, in <module>
    from frog import main
  File "/app/lib/python3.9/site-packages/frog/main.py", line 48, in <module>
    from .window import FrogWindow
  File "/app/lib/python3.9/site-packages/frog/window.py", line 42, in <module>
    class FrogWindow(Handy.ApplicationWindow):
  File "/app/lib/python3.9/site-packages/frog/window.py", line 45, in FrogWindow
    granite_settings: Granite.Settings = Granite.Settings.get_default()
gi.repository.GLib.GError: g-invoke-error-quark: Could not locate granite_settings_get_default: libgranite.so.6.2.0: cannot open shared object file: No such file or directory (1)
```

I don't really know how it is supposed to work but I see that file in `/app/lib`, and the application works correctly if I pass `--env=LD_LIBRARY_PATH=/app/lib` to Flatpak. I have a hunch that like Python bindings blindly hardcode /usr/lib, so you may need to add that `--env` line to `finish-args`.

**Comment by amka** [comment]
@barthalion yup found this. I gonna remove Granite.Settings completely today. So this won't be an issue.

**Comment by amka** [comment]
bot, build com.github.tenderowl.frog

**Pull request closed without merging**